I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 123, in script
    scene bg room
Exception: Image 'bg' does not accept attributes 'room'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 123, in script
    scene bg room
  File "C:\Users\hhall\Documents\apps\Setups\renpy-8.2.3-sdk\renpy\ast.py", line 1247, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "C:\Users\hhall\Documents\apps\Setups\renpy-8.2.3-sdk\renpy\ast.py", line 1076, in show_imspec
    renpy.config.show(name,
  File "C:\Users\hhall\Documents\apps\Setups\renpy-8.2.3-sdk\renpy\exports.py", line 739, in show
    if not base.find_target() and renpy.config.missing_show:
  File "C:\Users\hhall\Documents\apps\Setups\renpy-8.2.3-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "C:\Users\hhall\Documents\apps\Setups\renpy-8.2.3-sdk\renpy\display\displayable.py", line 246, in _duplicate
    args.extraneous()
  File "C:\Users\hhall\Documents\apps\Setups\renpy-8.2.3-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'bg' does not accept attributes 'room'.

Windows-10-10.0.26100 AMD64
Ren'Py 8.2.3.24061702
Chic Boutiqe Prototype v01 1.0
Mon Jan 27 13:49:34 2025
